src/css/categories.css

```
@@ -0,0 +1,38 @@
+.product-grid {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
+    gap: 20px; /* Spacing between grid items */
+    justify-content: center; /* Center horizontally */
+    align-items: center; /* Center vertically */
+    margin-left: 15%;
+    margin-right: 15%;
+}
+  
+.product-card {
+    display: flex;
+    flex-direction: column;
+    border: 1px solid #ccc;
+    padding: 10px;
+    text-align: center;
+    width: 200px;
+    min-height: 300px;
+    justify-content: center; /* Center horizontally */
+    align-items: center; /* Center vertically */
+}
+
+.product-card img {
+    width: 80%; /* Adjust the width as needed */
+    height: auto; /* Allow the height to adjust proportionally */
```

There's a thousand ways to skin this cat, however because images are so variable I think just specifying a max width/height will account for these variations better
```suggestion
.product-card img {
    max-width: 75%;
    max-height: 50%;
```